website/docs/d/storage_profile.html.markdown

```diff
@@ -32,5 +32,25 @@ The following arguments are supported:
 * `name` - (Required) Storage profile name.
 
 ## Attribute reference
+* `id` - storage profile identifier
 
-This data source exports only `id` field.
+(Supported in provider *v3.6+*)
+
+* `limit` - Maximum number of storage bytes (scaled by Units) allocated for this profile. A value of 0 is understood to mean `maximum possible`
+* `used_storage` - Storage used, in Megabytes, by the storage profile
```

Comment:
Maybe it is just me, but it sounds strange - could this be simpler.

```suggestion
* `used_storage` - Storage used, by the storage profile (in Megabytes)
```

Reply:
changed